{"name": "test_login_falhou", "status": "passed", "steps": [{"name": "Abrir a tela de login", "status": "passed", "start": 1755723949704, "stop": 1755723949704}, {"name": "Preencher usuário e senha incorretos", "status": "passed", "start": 1755723949704, "stop": 1755723949704}, {"name": "Clicar no botão de login", "status": "passed", "start": 1755723949704, "stop": 1755723949705}, {"name": "Validar mensagem de erro", "status": "passed", "attachments": [{"name": "Erro", "source": "a0813be4-739b-4e12-bcc4-5f424b5a0ba9-attachment.txt", "type": "text/plain"}], "start": 1755723949705, "stop": 1755723949705}], "attachments": [{"name": "stdout", "source": "26dadb28-4c2e-4741-b518-a88f8cc0efaa-attachment.txt", "type": "text/plain"}], "start": 1755723949704, "stop": 1755723949706, "uuid": "1e91da9a-7b1a-47be-8b2e-4156d192febd", "historyId": "7ad02c982fd228e2fb49f66a4359d1fa", "testCaseId": "7ad02c982fd228e2fb49f66a4359d1fa", "fullName": "tests.test_login#test_login_falhou", "labels": [{"name": "story", "value": "Falha no login com senha incorreta"}, {"name": "feature", "value": "Funcionalidade de Login"}, {"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "DESKTOP-9U5D9IN"}, {"name": "thread", "value": "15116-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}], "titlePath": ["tests", "test_login.py"]}